src/modules/linkedservices/keyvault.bicep

```diff
@@ -0,0 +1,36 @@
+// Source: 
+// Date: 2023-02-02
+// Version: 
+
+@description('Required. The name of the parent Azure Data Factory.')
+param dataFactoryName string
+
+@description('The name of the keyvault.')
+param keyVaultName string
+
+resource dataFactoryRef 'Microsoft.DataFactory/factories@2018-06-01' existing = {
+  name: dataFactoryName
+}
+
+resource linkedService 'Microsoft.DataFactory/factories/linkedservices@2018-06-01' = {
```

Comment:
nit: Given how small these files are, should we just put all linked services together?